src/components/organisms/Carousel/index.js

```diff
@@ -1,18 +1,10 @@
 import React from 'react';
-import CarouselText from '../../molecules/CarouselText';
-import Button from '../../atoms/Button';
+import Slider from '../../molecules/Slider';
 import './style.scss';
 
 const Carousel = ({ title, subtitle, images }) => (
```

Comment:
This entire component is just a wrapper for slider, get rid of it

Reply:
Deleted